app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -132,14 +138,37 @@ public static BrowserFragment create(String url) {
     // pending action for file-choosing
     private FileChooseAction fileChooseAction;
 
+    // monitor screenshot is created
+    private FileObserver fileObserver;
+
+    private AlertDialog dialogScreenshotOnBoarding;
+
     @Override
     public void onPause() {
         super.onPause();
+        if(fileObserver != null)
+            fileObserver.stopWatching();
     }
 
     @Override
     public void onResume() {
         super.onResume();
+        if(Settings.getInstance(getActivity()).shouldShowScreenshotOnBoarding()) {
```

Comment:
We need to ask for External Path permission to do this

Reply:
After discussion, we decide to not ask for permission for now so nothing to change here.

Reply:
This is still an issue but I think we can merge first and worry about this if our UX tried and would like to prompt for the permission once we starts the app. Prompt every time we are notified for a new entry does not make sense for me so I guess the only place reasonable will be when the app is started, that we be pretty simple if we want to do so so let's merge this first.